zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -872,8 +872,11 @@ public static Either<ProblemDetail, TenantMemberRequest> toTenantMemberRequest(
                 request.getElements().stream()
                     .map(
                         element ->
-                            new AdHocSubProcessActivateActivityReference(element.getElementId()))
-                    .toList()));
+                            new AdHocSubProcessActivateActivityReference(
+                                element.getElementId(),
+                                getMapOrEmpty(element, e -> e.getVariables())))
+                    .toList(),
+                request.getCancelRemainingInstances()));
```

Comment:
🔧 This would become

```suggestion
                request.getCancelRemainingInstances() == null ? false : request.getCancelRemainingInstances));
```